### ISSUE TITLE
PrimeNG: Refactor Register Form Component

### ISSUE THREAD

**Issue by Whamo12** [softrams/bulwark#375]
The Register form component, used for creating and updating forms, should be refactored to use [PrimeNG](https://www.primefaces.org/primeng/showcase/#/)'s form components so that the styling is consistent with the remainder of the application.

Relative path: `frontend/src/app/register`

**Comment by NOPROD** [comment]
On it after #387 ^^

**Comment by NOPROD** [comment]
old layout : 

mobile:
![old-register-mob](https://user-images.githubusercontent.com/38657472/96793467-bdaf4a00-13fc-11eb-9b25-660e27ab03ce.PNG)

desktop:
![old-register-desk](https://user-images.githubusercontent.com/38657472/96793470-be47e080-13fc-11eb-8a74-21f903b3176c.PNG)

new layout:

mobile:
![new-register-mob](https://user-images.githubusercontent.com/38657472/96793484-c9027580-13fc-11eb-8407-6661cc04239e.PNG)

desktop:
![new-register-desk](https://user-images.githubusercontent.com/38657472/96793486-c99b0c00-13fc-11eb-89d2-dfc62cd3fc56.PNG)

**Comment by Whamo12** [comment]
Wow, thank you so much for all the contributions.  We all appreciate it here! Looking forward to more collaborations! As I said, if you have any suggestions, please let us know.

**Comment by Whamo12** [comment]
>so i think i will finish all frontend issue with Prime NG 

@NOPROD If you have any ideas on how to do implement a dark mode theme, let us know in a new ticket. That's one of the final goals of moving over to PrimeNG since it has built in themes.
